package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@microsoft/globe",
-  "version": "2.6.0",
+  "version": "2.7.0",
```

Comment:
you havn't really changed contract, I think increasing just patch version is fine here